src/main/java/org/jenkinsci/plugins/codefresh/CFApi.java

```diff
@@ -107,8 +110,7 @@ public String getUser() throws MalformedURLException, IOException
         InputStream is = conn.getInputStream();
         String jsonString = IOUtils.toString(is);
         JsonObject user = new JsonParser().parse(jsonString).getAsJsonObject();
-        String userName = user.get("userName").getAsString();
-        return userName;
+        return user.get("userName").getAsString();
```

Comment:
defining variables for return values was done because of findbugs in maven release flow.
It will complain and fail if you return without putting the value in a local variable first